website/docs/getting-started/using.md

```diff
@@ -5,7 +5,7 @@ title: Using
 To use the Hook, first call it, telling it how many items your menu will have:
 
 ```jsx
-const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(numberOfItems);
+const { buttonProps, itemProps, isOpen, moveFocus, setIsOpen } = useDropdownMenu(numberOfItems);
```

Comment:
So with the getting started example, right now we've documented every variable it returns, but as we add more, this probably isn't the best strategy. Instead, it's probably best if these basic examples only document the variables you'd need to use for simple use cases.

With that in mind, I've created #283 to remove `setIsOpen` from these deconstructions, and I'd want to leave this file unchanged in this PR.

```suggestion
const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(numberOfItems);
```